src/countOccurrences.js

```diff
@@ -18,7 +18,17 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  let counter = 0;
+  const letterUp = letter.toUpperCase();
+  const phraseUp = phrase.toUpperCase();
```

Comment:
```suggestion
  const phraseUpperCase = phrase.toUpperCase();
```
не сокращай названия